cloudinary-image-management-and-manipulation-in-the-cloud-cdn/js/src/components/terms_order.js

```diff
@@ -1,113 +1,124 @@
 /* global window wp wpAjax */
 
-const Terms_Order = {
+export const Terms_Order = {
```

Comment:
Good idea to move the `export` up to here. It'd be easy to miss that, with it being in the middle of the file.